bulbs/videos/static/videos/js/series/video-detail-page/video-detail-page.js

```diff
@@ -0,0 +1,22 @@
+var SeriesVideoGrid = require('../series-video-grid/series-video-grid');
+
+VideoDetailPage = function() {
+  this.VideoDetailSeriesMeta = require('../video-detail-series-meta/video-detail-series-meta');
```

Comment:
We can move this up to the top of this file and we shouldn't need the class object set on `this`.  It probably makes more sense, if we need it to set the result of this line on `this`: 

```
new this.VideoDetailSeriesMeta(videoObject.series_slug);
```